### PR TITLE
Spanner GA.

### PR DESCRIPTION
**Pull request by benwulfe**
Also moves dependent library Google.Cloud.IAM.V1 to GA.
A subsequent checkin will switch the IAM dependency to "1.0.0" from "project" once everything is in nuget.

### CODE REVIEW

File: apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (jskeet) [inline apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml]: You can delete this file - if you fetch from master again, there's a change which will prevent this from being generated. (With this in place, the tests will fail to run.)

Comment (benwulfe) [reply apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml]: hmm its still being generated for me with a rebase from master.  I'm looking into it.

Comment (benwulfe) [reply apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml]: 25e0c80ac634172e6556b4d42286a6ec7cc6e9f2 suppresses generation of the csproj, but possibly not the coverage.xml.  I'll see about adding that.

**Comment by jskeet** [comment]
Do you have any C# files in that snippets directory? That's what it uses to check.

The commit has a comment starting "Add an effectively-empty snippets directory for EF" if that's any help.

File: apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.Snippets/coverage.xml]: Gah - really sorry about that. I must have not rerun it after thinking I'd fixed it :(

**Comment by benwulfe** [comment]
I've touched up the project generation tool to also stop coverage.xml generation using the same test you did earlier.  Appears the fix the problem.
I'll go ahead with this change Jon if I don't hear back from you.

**Pull request merged**